<?xml version="1.0"?>
<launch>
  <arg name="debug_mode" default="false"/>
  <arg name="prefix" value="gdb -ex run --args" if="$(arg debug_mode)"/>
  <arg name="prefix" value="" unless="$(arg debug_mode)"/>
  <node ns="fov_segmentation" pkg="cob_3d_fov_segmentation"
        type="fov_segmentation_node" name="fov_segmentation" output="screen" launch-prefix="$(arg prefix)">
    <remap from="shape_array_in" to="/geometry_map/map_array"/>
    <!-- param name="target_frame" type="string" value="/head_cam3d_link"/-->
    <!--param name="file_path" type="string" value="/Bag_Files/kitchen_table_chair"/>
    <param name="save_to_file" type="bool" value="false"/>
    <param name="map_frame_id" type="string" value="/map" />
    <param name="d" type="double" value="0.25" />
    <param name="cos_angle" type="double" value="0.93" />
    <param name="enable_tf" type="bool" value ="true" />
    <param name="enable_cyl" type="bool" value="true" />
    <param name="enable_poly" type="bool" value="true" /-->
  </node>
</launch>
